app/logger/logger_config_aws.yaml

```
@@ -29,4 +29,4 @@ root:
   level: WARNING
   handlers: [console_handler, aws_handler]
 
-disable_existing_loggers: False
+disable_existing_loggers: False
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)